developer/src/kmc/src/util/KeymanSentry.ts

```diff
@@ -112,4 +128,10 @@ This error has been automatically reported to the Keyman team.
       throw e;
     }
   }
+
+  static async close() {
+    if(isInit) {
+      await Sentry.close(2000);
```

Comment:
What is the 2,000?  milliseconds to timeout? Might be worth splitting to a constant to document what it is